eng/versioning/version_data.txt

```diff
@@ -23,10 +23,12 @@ com.microsoft.azure.cognitiveservices:azure-cognitiveservices-contentmoderator;1
 com.microsoft.azure.cognitiveservices:azure-cognitiveservices-customvision-prediction;1.1.0-beta.3;1.1.0-beta.3
 com.microsoft.azure.cognitiveservices:azure-cognitiveservices-customvision-training;1.1.0-beta.3;1.1.0-beta.3
 com.microsoft.azure.cognitiveservices:azure-cognitiveservices-faceapi;1.1.0-beta.1;1.1.0-beta.1
-com.microsoft.azure:azure-cosmos-parent;3.5.0;3.5.1-beta1
-com.microsoft.azure:azure-cosmos;3.5.0;3.5.1-beta1
-com.microsoft.azure:azure-cosmos-benchmark;3.5.1-beta1;3.5.1-beta1
-com.microsoft.azure:azure-cosmos-examples;3.5.1-beta1;3.5.1-beta1
+# azure-cosmos-parent doesn't have a dependency management section anymore, it doesn't need to change unless parent pom changes 
+com.microsoft.azure:azure-cosmos-parent;3.5.0;3.5.0
+com.microsoft.azure:azure-cosmos;3.5.0;3.6.0-beta.1
+# azure-cosmos-benchmark and azure-cosmos-examples don't get released, and should be set to azure-cosmos current-version
```

Comment:
Why set them to a moving target which is "azure-cosmos" current-version? Wouldn't it be easier if they were set to a fixed version?

Reply:
We have always kept them in sync with azure-cosmos version. That is the only reason to keep incrementing them so as to avoid confusion within the Cosmos SDK team. 

Once we publish v4, we will try to get rid of `azure-cosmos-parent`, `azure-cosmos-benchmark` and `azure-cosmos-examples` 

For now, we would like to keep them updated with azure-cosmos current version.